### PR TITLE
behavioral -> behavioUral

### LINKED ISSUE #5683: [Cleanup] behavioral -> behavioUral
as of Nov. 2019, we're keeping the U in `behaviour` and `behavioural` and would like to make this consistent across all module names / display text / docs

Fresh search links:
* https://github.com/aces/Loris/search?q=behavioral&unscoped_q=behavioral
* https://github.com/aces/Loris/search?q=behavior&type=Code

Biggest examples are probably: 
* Publication form option which users are asked to select  -> SQL patch needed?
*  Instrument List, bold table header: "Behavioral Battery..."

more innocuous examples exist (but should still be fixed) e.g.: 
- Statistics `Behavioral Feedback` etc...

### PR DESCRIPTION
**Pull request by haoweiqiu**
## Brief summary of changes

- [x] Have you updated related documentation?
The commits completely change all "behavior" to "behaviour".

Haowei Qiu, GSOC Applicant
CC @christinerogers 

#### Link(s) to related issue(s)

* Resolves #5683 (Reference the issue this fixes, if any.)

### CODE REVIEW

**Comment by haoweiqiu** [comment]
@christinerogers Thank you very much for your comment! The problem you mentioned has been resolved.

**Comment by christinerogers** [comment]
not urgent before 23 release.  Ready for re-review (johnsaigle) and testing (shen?), just to ensure it doesn't break anything.

File: htdocs/acknowledgements/acknowledgements.php [hunk 1]
```diff
@@ -6,7 +6,7 @@
  * PHP Version 5
  *
  * @category Loris
- * @package  Behavioral
+ * @package  behavioural
```
Comment (lingma) [inline htdocs/acknowledgements/acknowledgements.php:9]: It should have a capital first character

File: modules/dashboard/help/dashboard.md [hunk 2]
```diff
@@ -28,6 +28,6 @@ User-specific list of outstanding tasks such as data entry conflicts, incomplete
 
 Recently-uploaded documents for your study. Click on a document name to download the file directly from the Dashboard. Click **Document Repository** to view the complete list of available documents.
 
-#### Behavioral Feedback Notifications
+#### behavioural Feedback Notifications
```
Comment (lingma) [inline modules/dashboard/help/dashboard.md:31]: Capital first character

**Comment by christinerogers** [comment]
This will probably need a re-base onto master.

**Comment by lingma** [comment]
@christinerogers ? this is on the master

**Comment by ridz1208** [comment]
@driusan I fixed the conflicts, ready for your review

**Pull request merged**